manifests/istio-telemetry/kiali-operator/templates/serviceaccount.yaml

```diff
@@ -7,8 +8,9 @@ imagePullSecrets:
   {{- end }}
   {{- end }}
 metadata:
-  name: kiali-service-account
-  namespace: {{ .Release.Namespace }}
+  name: kiali-operator-service-account
+  namespace: {{ .Values.kialiOperator.namespace }}
```

Comment:
I think .Release.Namespace here refers to specifically to the kiali release, I could be wrong though

Reply:
Note: We want this (the Kiali Operator) to go into its own namespace, not where Kiali itself is to be deployed. This is analogous to how [Operator Lifecycle Manager (OLM) will install the operator](https://operatorhub.io/operator/kiali). This allows you to have a single operator install multiple Kialis (for multi-tenant scenarios, for example).